content/coordination/onboarding/index.en.md

```diff
@@ -143,29 +143,29 @@ we can add them to our records."
 
 ## Appendix B
 
-> confirm list of organisers - email template
+> confirm list of organizers - email template
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reply:
```suggestion
> confirm list of organisers - email template
```